utils/src/main/kotlin/Extensions.kt

```diff
@@ -207,6 +212,11 @@ fun File.searchUpwardsForSubdirectory(searchDirName: String): File? {
     return currentDir
 }
 
+/**
+ * Get the size of this [File] in mebibytes (MiB) with two decimal places as [String].
+ */
+val File.sizeInMiB: String get() = "${length().bytesToMiB().format()} MiB"
```

Comment:
In other existing code we use "Mib" as a suffix, but you start to use "MiB". While I agree that "MiB" maybe reads nicer, I believe according to Java naming conventions we should still use "Mib". In any case, we should align on the spelling either way.

Also, the property name sounds to me as if it would hold a number, not a string, and without a " MiB" suffix. So how about naming the property something like "sizeAndUnitInMib" (or so) instead?

Reply:
Done. Used `formatSizeInMib`.